staking/handler_test.go

```diff
@@ -148,12 +148,12 @@ func (suite *HandlerTestSuite) TestHandleMsgValidatorUpdate() {
 
 	ValFrmID, ok := keeper.GetValidatorFromValID(ctx, oldSigner.ID)
 	require.True(t, ok, "new signer should be found, got %v", ok)
-	require.Equal(t, ValFrmID.Signer.Bytes(), newSigner[0].Signer.Bytes(), "New Signer should be mapped to old validator ID")
+	require.NotEqual(t, ValFrmID.Signer.Bytes(), newSigner[0].Signer.Bytes(), "Should not update state")
```

Comment:
Shouldn't we check for Equality with `oldSigner.Signer.Bytes()`

Reply:
Its same as we fetched `ValFrmID` from `oldSigner.ID`